src/mscorlib/shared/System/Globalization/IdnMapping.cs

```diff
@@ -156,6 +157,14 @@ public override int GetHashCode()
             return (_allowUnassigned ? 100 : 200) + (_useStd3AsciiRules ? 1000 : 2000);
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private static unsafe string GetStringForOutput(string originalString, char* input, int inputLength, char* output, int outputLength)
```

Comment:
optional: might make sense to pass ReadOnlySpan directly so that we have less unsafe methods

Reply:
Thanks.  I think we should look at changing most of the pointers to be spans in the implementation.  For now, though, since we already have the pointers, I'm going to leave this as is.